lahja/base.py

```diff
@@ -200,3 +193,86 @@ def __reduce__(self) -> None:  # type: ignore
         event = await agen.asend(None)
         await agen.aclose()
         return event
+
+    def is_remote_subscribed_to(
+        self, remote_endpoint: str, event_type: Type[BaseEvent]
+    ) -> bool:
+        """
+        Return ``True`` if the specified remote endpoint is subscribed to the specified event type
+        from this endpoint. Otherwise return ``False``.
+        """
+        for endpoint, subscriptions in self.get_connected_endpoints_and_subscriptions():
+            if endpoint != remote_endpoint:
+                continue
+
+            for subscribed_event_type in subscriptions:
+                if subscribed_event_type == event_type:
```

Comment:
`is` checks for direct class comparison are slightly more correct since classes are singletons.